pallets/clusters/src/lib.rs

```diff
@@ -29,40 +29,50 @@ pub struct ClusterMember {
 /// The **settings** of a **Role**
 #[derive(Encode, Decode, Clone, PartialEq, Debug, Eq, scale_info::TypeInfo)]
 pub struct RoleSettings {
+	/// Name of the setting
 	pub name: Vec<u8>,
+	/// The data associated to the Role
 	pub data: Vec<u8>,
 }
 
 /// **Struct** of **Role** belonging to a **Cluster**.
 #[derive(Encode, Decode, Clone, PartialEq, Debug, Eq, scale_info::TypeInfo)]
 pub struct Role {
+	/// Name of the role
 	pub name: Vec<u8>,
+	/// Settings of the Role
 	pub settings: RoleSettings,
-	pub members: Vec<ClusterMember>,
+	/// The list of Members associated to the role
+	pub members: Vec<Hash256>,
+	/// The optional list of Rules associated to the Role
 	pub rules: Option<Rule>,
 }
 
 /// **Struct** of **Rule** belonging to a **Role**.
 #[derive(Encode, Decode, Clone, PartialEq, Debug, Eq, scale_info::TypeInfo)]
 pub struct Rule {
+	/// The name of the Rule
 	pub name: Vec<u8>,
 }
 
 /// **Struct** of a **Cluster**
 #[derive(Encode, Decode, Clone, PartialEq, Debug, Eq, scale_info::TypeInfo)]
 pub struct Cluster<TAccountId> {
+	/// The owner of the Cluster
 	pub owner: TAccountId,
+	/// The name of the Cluster
 	pub name: Vec<u8>,
+	/// The list of Roles associated to the Cluster
 	pub roles: Vec<Hash256>,
+	/// The list of Members of the Cluster
 	pub members: Vec<Hash256>,
```

Comment:
what's hash256?, this should be AccountID no?

Reply:
It was `Vec<Hash256>` because I had a dedicated struct for `Member` that also included `data`, but now I removed it because it is not clear what a `data` field could mean for a member at the moment (also considering that it was a `Vec<u8>`, so it also needed a limit).
So, now the cluster has the list of members as [`Vec<T::AccountId>`](https://github.com/fragcolor-xyz/clamor/blob/8276f14d8e8d6eb42a1fcc163adbc348619888ed/pallets/clusters/src/lib.rs#L62).